test/desktop/features/scripts/build_maze_runner_web_gl.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
At some point I'd look to unify this with the existing script, as there is a lot of commonality.  I wouldn't do it just yet, though, as I'm making some updates to it for Windows.  Indeed, I'll probably have cause to look at this script when I come to add the MazeRunner steps, so I could probably look at unifying it at that point.

Reply:
in the end i think we would probably have one script that takes arguments. ./build_maze_runner --windows etc.